src/gui/src/components/explorer-grid/selected-item/tabs-overview.tsx

```diff
@@ -15,14 +24,80 @@ export const OverviewTabButton = () => {
 }
 
 export const OverviewTabContent = () => {
-  const { selectedItemDetails, selectedItem } = useSelectedItem()
+  const { manifest } = useSelectedItem()
```

Comment:
I'd rather normalize homepage, funding, bugs and keyword values in dedicated helper methods so that the template here doesn't need so many conditionals. On the plus side these helper methods tend to be simple and fun to unit test.

```suggestion
  const { manifest } = useSelectedItem()
  const homepage: string = getHomepage(manifest)
  const funding: string[] = getFunding(manifest)
  const bugs: { url?: string, email?: string} = getBugs(manifest)
  const keywords: string[] = getKeywords(manifest)
```

These helpers should probably live in `@vltpkg/types` but anyways, not something that needs to be addressed in this PR. It can be a later refactor.

Reply:
I like this a lot, we could definitely create a bunch of helpers later on